### PR TITLE
Add draft of history.

### PR DESCRIPTION
**Pull request by joshwalawender**
This is a draft article on the history of PANOPTES.  This was written a few months ago and needs a proper review and editing.

### CODE REVIEW

File: history.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Automation
+
+Olivier constructed a prototype automated camera system, but could only obtain limited data from home due to poor weather.  Olivier, like many of the original PANOPTES team members lives in Hilo, Hawaii -- the rainiest city in the United States.  Some areas of Hilo average over [175 inches (4.56 meters)](http://rainfall.geography.hawaii.edu/interactivemap.html) of rain per year.
+
+At the time, another eventual PANOPTES team member, Josh Walawender, was working on a different small robotic telescope project which was located at the [Mauna Loa Observatory](https://www.esrl.noaa.gov/gmd/obop/mlo/) run by NOAA on the slopes of Mauna Loa.  With his help, the original test unit of PANOPTES was hosted by MLO and infrastructure (power and networking) was provided through the VYSOS project.
```
Comment (oguyon) [inline history.md]: MLO and VYSOS acronyms should be defined
include webpage for VYSOS, or briefly mention what it's about

File: history.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Based on experience at Mauna Loa, the hardware of PANOPTES was tested and refined.  We were able to demonstrate that low cost hardware could be made to operate reliably under adverse weather conditions even without a protective dome or roof to protect the equipment.
```
Comment (oguyon) [inline history.md]: We -> "Josh and Olivier" 
or ...
include small statement before this paragraph "Others joined the project and formed the initial PANOPTES team", then OK to use "We"

File: history.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+## Science
+
+The science goal of PANOPTES is to survey moderately bright stars (roughly 9th to 12th magnitude) to look for exoplanet transits.  These stars are much brighter than the typical star studied by [NASA's Kepler mission](https://en.wikipedia.org/wiki/Kepler_(spacecraft)) and thus are likely to be closer to Earth which should make those planets easier to separate from their star.  Some of the PANOPTES team also work on extreme adaptive optics systems such as the [SCExAO project at Subaru Telescope](https://www.subarutelescope.org/Projects/SCEXAO/) which can be used to image nearby planetary systems, so the link between these two interests is that PANOPTES can find targets that extreme AO systems (like SCExAO) can then follow up on.
```
Comment (oguyon) [inline history.md]: suggested addition after Kepler statement:
"Thanks to its highly distributed network of units, PANOPTES can provide nearly continuous observations using units around the globe, and observe large parts of the sky. This flexibility complements professional transit surveys that use a smaller number of telescopes, such as Kepler or TESS [link]."

Last statement can be broader:
"PANOPTES will find new exoplanets that can be studied in detail with larger telescopes using techniques such as radial velocity, transit spectroscopy, and direct imaging. In fact, some of the PANOPTES team also work on ..."

**Comment by joshwalawender** [comment]
I've made some updates based on @oguyon 's comments.

File: history.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+This color pixel array normally makes determining the brightness of a star difficult because of the star's small size.  The majority of the light for in focus stars falls on to a single pixel, so that star will appear different brightness depending on whether it happened to fall on a red, green, or blue pixel.
+
+To get around this, Olivier's algorithm generates a comparison stellar image baed on selecting other stars in the image which fall on the same position in the RGGB color filter pattern and which are similar in shape (and other properties) to the star being measured.  By watching how stars with the same "errors" due to the color filter array (and other effects) change in brightness due to those "errors" over many images, we can compare their behavior to the star being measured and see if it has brightened or faded during a particular image sequence.
```
Comment (jamessynge) [inline history.md]: Change baed to based.

**Comment by joshwalawender** [comment]
Apologies for the long line length.  I'm not a very linear writer, so if I set the line lengths manually, it rapidly becomes a mess as I go back and edit pervious lines.  Is there an automated way to insert the line breaks from my raw one line per paragraph file?

**Comment by jamessynge** [comment]
FYI, I reformatted it using a tool in my editor. Github's web based editor doesn't offer that.

**Pull request merged**